v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
```

Comment:
@jseldess Found this in the test file, I think this example would work:
SELECT (ARRAY['a', 'b', 'c'])[2]
----
b

In the similar vein, can we add an example for the append (with ||) functionality? I know we normally don't include examples of functions in the data type docs, but can we make an exception?